azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -203,16 +205,21 @@ public void run() throws Exception {
     String effectiveUser = null;
     final boolean isExecuteAsUser = this.sysProps.getBoolean(EXECUTE_AS_USER, true);
 
+    //Get list of users we never execute flows as. (ie: root, azkaban)
+    final String [] userBlackList = this.sysProps.getString(BLACK_LISTED_USERS, "root,azkaban").split(",");
```

Comment:
If this is a new configuration key, then you can add it to `Constants.ConfigurationKeys` instead. 

We don't yet have a good standard for placing defaults. Some are listed as `DEFAULTS_*` in `Constants class like `DEFAULT_SSL_PORT_NUMBER`.